pgbulk/core.py

```diff
@@ -75,12 +104,15 @@ class UpdateField(UserString):
         )
     """
 
-    def __init__(self, field: str, expression: Union[models.Expression, None] = None):
-        self.data = field
-        self.expression = expression
+    expression: Expression
 
+    def __new__(cls, field: str, expression: Expression) -> "UpdateField":
```

Comment:
Not sure if I understand the need to inherit `str`. My understanding is that it's generally desired to inherit `UserString` for things like this.

The original intention of `UpdateField` was to just be a single string field name, hence why the expression was optional. What types of issues did you run into with having the expression be optional?

I doubt anyone is using it this way, so it's probably safe to do this without disruption. Mostly just wondering why

Reply:
`UserString` preexisted being able to directly inherit from `str`. While `UserString` is convenient, one issue is that is not treated as string, without extracting `data` from it, so simply inheriting from `str` makes things simpler from a type-safety perspective.

>>> The original intention of UpdateField was to just be a single string field name, hence why the expression was optional. What types of issues did you run into with having the expression be optional?

After taking another look, I think I can continue having it be nullable, let me make a quick commit.

Reply:
Okay, added back nullability.